scripts/github-status/github-status.rb

```diff
@@ -14,6 +21,19 @@ def initialize(config = {})
     @client = Octokit::Client.new(:netrc => true)
     @client.auto_paginate = true
     @client.login
+    @members = {}
+  end
+
+  def team_members(team_name)
+    # only query when needed
+    unless @members[team_name] then
```

Comment:
yes it is and its tested
`unless` is just `if not`, so the code says `if not ... then`